src/SprykerEco/Yves/Unzer/UnzerFactory.php

```diff
@@ -40,6 +56,26 @@ public function createMarketplaceCreditCardFormDataProvider(): StepEngineFormDat
         );
     }
 
+    /**
+     * @return \Spryker\Yves\StepEngine\Dependency\Form\StepEngineFormDataProviderInterface
+     */
+    public function createBankTransferFormDataProvider(): StepEngineFormDataProviderInterface
+    {
+        return new BankTransferFormDataProvider(
+            $this->getQuoteClient(),
+        );
+    }
+
+    /**
+     * @return \Spryker\Yves\StepEngine\Dependency\Form\StepEngineFormDataProviderInterface
+     */
+    public function createSfortFormDataProvider(): StepEngineFormDataProviderInterface
```

Comment:
Please, fix the method's name `createSofortFormDataProvider()`

Reply:
Fixed.